<ui version="4.0" >
 <class>EpgBrowserClass</class>
 <widget class="QMainWindow" name="EpgBrowserClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Browser</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/icons/images/browser.png</normaloff>:/icons/images/browser.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QHBoxLayout" name="headerLayout" >
      <item>
       <widget class="QLineEdit" name="addressLine" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonGo" >
        <property name="minimumSize" >
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="icon" >
         <iconset resource="images.qrc" >
          <normaloff>:/icons/images/go.png</normaloff>:/icons/images/go.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWebView" name="epgView" >
      <property name="url" >
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Toolbar</string>
   </property>
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome" />
   <addaction name="actionEpg" />
   <addaction name="separator" />
   <addaction name="actionBack" />
   <addaction name="actionForward" />
   <addaction name="actionReload" />
   <addaction name="actionStop" />
   <addaction name="separator" />
   <addaction name="actionClose" />
  </widget>
  <action name="actionHome" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/home.png</normaloff>:/icons/images/home.png</iconset>
   </property>
   <property name="text" >
    <string>Home</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileclose.png</normaloff>:/icons/images/fileclose.png</iconset>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionEpg" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/siol.gif</normaloff>:/icons/images/siol.gif</iconset>
   </property>
   <property name="text" >
    <string>SiOL TV EPG</string>
   </property>
  </action>
  <action name="actionReload" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/reload.png</normaloff>:/icons/images/reload.png</iconset>
   </property>
   <property name="text" >
    <string>Reload</string>
   </property>
  </action>
  <action name="actionBack" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/back.png</normaloff>:/icons/images/back.png</iconset>
   </property>
   <property name="text" >
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/forward.png</normaloff>:/icons/images/forward.png</iconset>
   </property>
   <property name="text" >
    <string>Forward</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/stop.png</normaloff>:/icons/images/stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>EpgBrowserClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>epgView</sender>
   <signal>titleChanged(QString)</signal>
   <receiver>EpgBrowserClass</receiver>
   <slot>setWindowTitle(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>399</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBack</sender>
   <signal>activated()</signal>
   <receiver>epgView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionForward</sender>
   <signal>activated()</signal>
   <receiver>epgView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReload</sender>
   <signal>activated()</signal>
   <receiver>epgView</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>activated()</signal>
   <receiver>epgView</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
